pkg/debug/scripts/collect-info.sh

```diff
@@ -20,6 +20,7 @@ INFO_DIR="eve-info-v$VERSION-$DATE"
 TARBALL_FILE="/persist/$INFO_DIR.tar.gz"
 SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
 
+READ_LOGS_DAYS=
```

Comment:
Please increase the VERSION variable after making changes in this script.
(probably to 26 if we merge this sooner: https://github.com/lf-edge/eve/pull/4095/files)

Reply:
I thought i did, but i guess someone upgraded before this pr, and i missed that. will do.

Reply:
You'll need to increase it again since I merged a PR to collect-info.sh today

Reply:
ok. Changed.